AzureData/README.md

```diff
@@ -119,12 +119,24 @@ database.get (collectionWithId: id) { r in
 
 #### Delete
 ```swift
-AzureData.delete (collection, from: databaseId) { s in
-    // s == successfully deleted
+AzureData.delete (collection, fromDatabase: databaseId) { r in
+    // successfully deleted == r.isSuccess
 }
 
-database.delete (collection) { s in
-    // s == successfully deleted
+AzureData.delete (collectionWithId: collectionId, fromDatabase: databaseId) { r in
+    // successfully deleted == r.isSuccess
+}
+
+database.delete (collection) { r in
+    // successfully deleted == r.isSuccess
+}
+
+database.delete (collectionWithId: collectionId, fromDatabase: databaseId) { r in
```

Comment:
Don't think this override is necessary, if we have the `database` object, we should be able to get e the id from there, shouldn't have to pass it in manually.   (if this is the way it is in Android, we should let them know)

Reply:
Oh! Yes. I actually haven't implemented this override. Just made a mistake while updating the README.